lib/narwhal.js

```diff
@@ -39,6 +40,13 @@ narwhal.prototype.updateNarwhal = function() {
       this.y -= this.speed * (Math.cos(this.angle));
       this.tuskY -= this.speed * (Math.cos(this.angle));
     }
+    if (this.burst){
+      this.x += (this.speed *20) * (Math.sin(this.angle));
+      this.tuskX += (this.speed *20) * (Math.sin(this.angle));
+      this.y -= (this.speed *20) * (Math.cos(this.angle));
+      this.tuskY -= (this.speed *20) * (Math.cos(this.angle));
+      this.burst = false;
+    }
```

Comment:
Added burst function. Currently the narwhal will jump 20 px. we need to work on how to have him move 1px at a time but quicker